### ISSUE TITLE
Standalone json output not working on arcguard scanner cli all 2.2.2

### ISSUE THREAD

**Issue by JayGhiya** [archguard/archguard#153]
**Describe the bug**
Standalone output in json is not working as of now post upgradation to 2.2.2. The same command works with 2.1.5.
```bash
java --add-opens=java.base/java.nio=ALL-UNNAMED -jar /Users/jayghiya/Documents/unoplat/scanner_cli-2.2.2-all.jar --with-function-code --path=/Users/jayghiya/Documents/unoplat/textgrad/ --output=json --language=python --output=arrow --depth=10
```
**Env (please complete the following information):**
 - OS: Macos
 - Env: Local Dev
 - Browser Not applicable 

**To Reproduce**
Run the command for python codebase for https://github.com/zou-group/textgrad and it does not work.

**Expected behavior**
Ideally 0_Codes.json should be having all the metadata but in this case it creates an empty file. 2.1.5 works fine.

**Comment by phodal** [comment]
@JayGhiya thanks for feedback, already fixed it, try 2.2.3

**Comment by JayGhiya** [comment]
@phodal thanks :) much appreciated! tried and it worked
